common/app/views/fragments/atoms/interactive.scala.html

```diff
@@ -0,0 +1,44 @@
+@(interactive: _root_.model.content.InteractiveAtom, shouldFence: Boolean)
+
+@body = {
+    <style>
+        @HtmlFormat.raw(interactive.css)
+    </style>
+    @HtmlFormat.raw(interactive.html)
+}
+
+@iframeBody = {
+    @body
+    <script>
+        function resize() {
+            window.frameElement.height = document.body.offsetHeight;
+        }
+
+        var fonts = [].slice.apply(window.parent.document.styleSheets)
+            .filter(function (sheet) { return sheet.ownerNode.className.indexOf("webfont") > - 1; })
+            .map(function (sheet) { return sheet.ownerNode.textContent; })
+            .join(' ');
+        var css = document.createElement('style');
+        css.textContent = fonts;
+        document.head.appendChild(css);
+
+        window.addEventListener('resize', resize);
+        resize();
+    </script>
+    @for(js <- interactive.mainJS) {
+        <script src="https://interactive.guim.co.uk/libs/require.js/2.3.2/require.min.js"></script>
+        <script>
+            require(['@js'], function (interactive) {
+                interactive.boot(document.body);
+            });
+        </script>
+    }
+}
+
+@if(shouldFence) {
+    <iframe class="interactive-atom" srcdoc="@iframeBody.toString"></iframe>
```

Comment:
So, we're not fencing here, right? My concern would be that `srcdoc` isn't that well supported:

![image](https://cloud.githubusercontent.com/assets/638051/19345902/9761b7f2-9138-11e6-95c0-1156037c7b37.png)

So what is your plan for handling IE11, Edge if not using fence.js? I'm surprised it didn't handle resizing of iframes by content, I don't remember interactive embeds always having scroll bars?